apps/mobile/src/features/send-refactor/screens/select-asset.tsx

```diff
@@ -0,0 +1,65 @@
+import { FullHeightSheetHeader } from '@/components/full-height-sheet/full-height-sheet-header';
+import { FullHeightSheetLayout } from '@/components/full-height-sheet/full-height-sheet.layout';
+import { HeaderBackButton } from '@/components/headers/components/header-back-button';
+import { AssetPicker } from '@/features/send-refactor/components/asset-picker/asset-picker';
+import { usePreloadBtcData } from '@/features/send-refactor/hooks/use-preload-btc-data';
+import { usePreloadStxData } from '@/features/send-refactor/hooks/use-preload-stx-data';
+import { useSendNavigation, useSendRoute } from '@/features/send-refactor/navigation';
+import { useSendFlowContext } from '@/features/send-refactor/send-flow-provider';
+import { SendableAsset } from '@/features/send-refactor/types';
+import { NetworkBadge } from '@/features/settings/network-badge';
+import { i18n } from '@lingui/core';
+import { t } from '@lingui/macro';
+
+export function SelectAsset() {
+  const { navigate } = useSendNavigation();
+  const route = useSendRoute<'select-asset'>();
+  const canGoBack = route.params?.previousRoute === 'select-account';
+  const {
+    state: { selectedAccount },
+    selectAsset,
+  } = useSendFlowContext();
+  // Preload relevant data to ensure smooth transition animation to form screen
```

Comment:
Is this to get around other lags we have? 

I'm not sure if it's good to always preload data from a different chain if we won't always use it?